x/domain/controllers/account/account.go

```diff
@@ -325,34 +325,34 @@ func (a *Account) deletableBy(addr sdk.AccAddress) error {
 	return nil
 }
 
-// ValidTargets verifies that the provided targets are valid for the account
-func ValidTargets(targets []types.BlockchainAddress) ControllerFunc {
+// ValidResources verifies that the provided resources are valid for the account
+func ValidResources(resources []types.Resource) ControllerFunc {
 	return func(ctrl *Account) error {
-		return ctrl.validTargets(targets)
+		return ctrl.validResources(resources)
 	}
 }
 
-// validTargets validates different blockchain targets address and ID
-func (a *Account) validTargets(targets []types.BlockchainAddress) error {
+// validResources validates different resources
+func (a *Account) validResources(resources []types.Resource) error {
 	a.requireConfiguration()
-	validBlockchainID := regexp.MustCompile(a.conf.ValidBlockchainID)
-	validBlockchainAddress := regexp.MustCompile(a.conf.ValidBlockchainAddress)
-	// create blockchain targets set to identify duplicates
-	sets := make(map[string]struct{}, len(targets))
-	// iterate over targets to check their validity
-	for _, target := range targets {
-		// check if blockchain ID was already specified
-		if _, ok := sets[target.ID]; ok {
-			return sdkerrors.Wrapf(types.ErrInvalidBlockchainTarget, "duplicate blockchain ID %s", target.ID)
+	validBlockchainID := regexp.MustCompile(a.conf.ValidResourceURI)
```

Comment:
`validBlockchainID` should be `validURI`

Reply:
@fdymylja , `ValidResourceURI` should be `ValidURI`, otherwise - there's redundancy on "resource".